assembler/styles/typography/01-messapia-bold-bricolage.json

```diff
@@ -1,7 +1,15 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
+	"version": 3,
```

Comment:
I ran the file through a validator with the version 3 schema, and the only error was the `categories` (as expected).